sklearn/linear_model/_logistic.py

```diff
@@ -1051,18 +1053,26 @@ class LogisticRegression(LinearClassifierMixin,
         l2 penalty with liblinear solver. Prefer dual=False when
         n_samples > n_features.
 
+        |TuningParameter|
+
     tol : float, default=1e-4
         Tolerance for stopping criteria.
 
+        |TuningParameter|
+
     C : float, default=1.0
         Inverse of regularization strength; must be a positive float.
         Like in support vector machines, smaller values specify stronger
         regularization.
 
+        |TuningParameter|
+
     fit_intercept : bool, default=True
         Specifies if a constant (a.k.a. bias or intercept) should be
         added to the decision function.
 
+        |ControlParameter|
```

Comment:
This as well. It does not influence the shape of the output, for example.

Reply:
And what of class_weight?

Reply:
I would say that one is hard to tune, like, I have not heard about one tuning that with exact values for classes. But for `"balanced"` vs. `None`, I think I would say you could try your pipeline with both and see what is the best, no?